app/src/Liquidator.ts

```diff
@@ -367,6 +416,19 @@ export default class Liquidator {
       };
     }
   }
+  
+  private async calculateBorrowerHealth(borrower: string): Promise<BorrowerHealthResult> {
+    const healthResult = await this.marginAccountLensContract.methods.getHealth(borrower, true).call();
+    const [healthAStr, healthBStr]: [string, string] = [healthResult[0], healthResult[1]];
```

Comment:
It's odd that these are strings and not BigNumbers

Reply:
indeed